scripts/sysman/memory.yml

```diff
@@ -132,6 +132,21 @@ members:
       desc: "[out] The total allocatable memory in bytes (can be less than the `physicalSize` member of $s_mem_properties_t)"
 --- #--------------------------------------------------------------------------
 type: struct
+desc: "Extension properties for Memory State"
+version: "1.7"
+class: $sMemory
+name: $s_mem_ext_state_t
+details:
+    - "Memory Page Offline Metrics"
```

Comment:
Please share if we should consider adding Page size as well ?

Reply:
Nope. Pagesize might be platform and memory config dependent which we can publish as specs.

Reply:
@aravindksg  @vaidyan1-int  @sanilkumar0   i think page size is important to update to UMD, it might be different per GPU going ahead ? Isn't it ?

Reply:
UMD doesnt need to know this information. Page allocation is handled by KMD and the application need not know any insights into number of offlined pages or page size